README.md

```diff
@@ -0,0 +1,701 @@
+
+In the case of any `write` capabilities expiring, a peer can choose to keep already accepted operations which rely on this capabilities authority after expiry has passed, or where possible, roll back these changes and remove the operations. Which approach is taken will depend on the desired behavior for the document.
+
+## Condition Attenuation
+
+Delegated conditions must not expand the scope of the capability it delegates from, it must only attenuate.
+
+The following table gives some examples of valid and invalid delegated conditions.
+
+| Received Conditions                                | Delegated Conditions                               | Valid | Reason                                        |
+| -------------------------------------------------- | -------------------------------------------------- | ----- | --------------------------------------------- |
+| `{document_ids: ["0X01", "0X02"]}`                 | `{document_ids: ["0X01"]}`                         | ✅    | Condition is attenuated                       |
+| `{schema_ids: ["events"]}`                         | `{schema_ids: ["events"], document_ids: ["0X01"]}` | ✅    | New condition added which restricts authority |
+| `{from_timestamp: 10, to_timestamp: 100}`          | `{from_timestamp: 50, to_timestamp: 80}`           | ✅    | Condition is attenuated                       |
+| `{schema_ids: ["events"], document_ids: ["0X01"]}` | `{schema_ids: ["events"]}`                         | ❌    | Condition removed                             |
+| `{document_ids: ["0X01"]}`                         | `{document_ids: ["0X01", "0X02"]}`                 | ❌    | Condition expanded                            |
+| `{from_timestamp: 50, to_timestamp: 80}`           | `{from_timestamp: 0, to_timestamp: 100}`           | ❌    | Condition expanded                            |
+
+## Principal Alignment
+
+In delegation, the `receiver` field of every proof must match the `issuer` field of the capability being delegated to. This alignment must form a chain back to the `subject` for each resource.
+
+## Issuer Validation
+
+The issuer field must match against the public key from the operation header.
+
+# Authorization
+
+Valid capabilities are used to authorize two main classes of action: read and write. Read actions result in operations being sent to other peers, while write actions involve a peer accepting and applying operations. This occurs during a sync session between two peers or when a client is making requests to a shared node.
+
+## Read authority
+
+Read requests are considered authorized if a valid read capability exists where the requesting peer is the `receiver` of the capability and the document being syncronised is included in it's authority scope. Only operations within the `from_timestamp` - `to_timestamp` range must be included.
```

Comment:
tiny nit, I initially mis-parsed this "from minus to", which is obviously nonsense, how about "from to to"?:

```suggestion
Read requests are considered authorized if a valid read capability exists where the requesting peer is the `receiver` of the capability and the document being syncronised is included in it's authority scope. Only operations within the `from_timestamp` to `to_timestamp` range must be included.
```

Reply:
Thanks 🙏 that was such a bad character choice 😅